controllers/iamrole_controller.go

```diff
@@ -390,6 +383,17 @@ func (r *IamroleReconciler) UpdateMeta(ctx context.Context, iamRole *iammanagerv
 	}
 }
 
+//GenerateRoleName returns a roleName that should be created in IAM
+func GenerateRoleName(iamRole iammanagerv1alpha1.Iamrole, props config.Properties) string {
```

Comment:
Can you actually move this to utils/utils.go file?

Reply:
:+1: